Automattic/wp-calypso
# Upsell higher Akismet tier in Calypso (i1)

Related to # 1201210512993648-as-1204165400433577/f

## Proposed Changes
In order to provide an Upgrade path for Akismet products we should enable Upgrade button in Calypso Purchases page. As a first iteration, we propose to directly navigate user to the next available plan at checkout, to avoid lengthy implementation and design.

## Testing Instructions
- Apply the branch locally and spin `yarn start` OR use Live link provided below
- Navigate to `/checkout/akismet/test` (if using live link provide a Feature flag param `?flags=akismet/siteless-checkout`) and purchase 10k plan
- Navigate to `/me/purchases` and select the new Purchased Akismet plan
![TOGIT](https://user-images.githubusercontent.com/60262784/227586286-435aaf81-35bf-4f8a-bda9-6409ecafd8a6.jpg)
- Hit the`Upgrade` button. You should be redirected to checkout for 20K plan. 
- You can repeat until the 40K plan, after which, it will redirect you to `https://akismet.com/enterprise`. Note, that in current Akismet site there's no such page, but you could update the hosts file with new one to test. 



## Pre-merge Checklist
